src/main/osd_slave/osd_slave_init.h

```diff
@@ -17,8 +17,13 @@
 
 #pragma once
```

Comment:
config_profile.h was deleted, not renamed - git is tracking this change incorrectly.